### ISSUE TITLE
Page Settings Bug: Title and Template and Custom CSS misconfigured and cannot be saved

### ISSUE THREAD

**Issue by optimizewp** [pojome/elementor#2178]
I worked on a site in preproduction phase before Elementor upgrade to v1.6 and higher, and know it is published. 
I noticed that every Template present in the library, built before the update, or created right now, when opened in the Elementor editor does not reflects the Page Settings Title and Template assigned in the WP Editor page before launching Elementor Editor. I found the bug because it is impossible to have the View Page working, it is stuck and opens another page instead, let's call it "Buggy" just for the sake of the explanation, always the same for different Templates.
I have a preproduction site still alive, with Elementor v1.5.5 where this problem is not present. I have a pre-release staging site in which I test the plugin updates before applying to the live site, and the live site, each one updated to Elementor 1.6.4 and the problem is present in each of these two sites. If I save an old page right now, it get assigned the same "Buggy" page title, which is related to a product page.
Another problem is with pages with Custom CSS added in the Page Settings Style Custom CSS section. The code is present in the database, but it is not retrieved and showed in the opened template/page. If I save the page it takes the Buggy title, the Template is read as Default, but let's say I used Elementor Canvas, it maintains Elementor Canvas in the DB, and the Custom CSS remains in DB. But not being able to see the Custom CSS code, it is easy to add new one and overwrite the old working one. This is a big problem.
Another test just made:
I create a brand Template, with title "Test page setting" slug "test-page-setting" Template "Elementor Canvas" saved and checked the related _posts and _postmeta tables.
Then edited the first time with Elemntor Editor, without any addition and saved. The corrects fields have been added in the _postmeta table. The Page Settings are showing "Buggy" as title and Default as Template.
I changed the title into FIXED and saved. The page title changed, in the DB too.
I reloaded the Elementor Editor page and there it is, the "Buggy" title again. Saving now does not changes the FIXED title assigned.
I changed then the Template to FW Fullwidth and saved. Title changed into "Buggy" and template into FW Fullwidth.
Changed status into Draft, saved. This is saved too, status changed.
Reloaded Elementor Editor: Again, Buggy, Default, Published. It does not read the current settings.

This could be an isolated problem related to this specific installation, if this is the case, I can give access to the staging site.
But I want to highlight once again that the same content, and installation setting, with Elementor 1.5.5 is working fine.

**Comment by optimizewp** [comment]
I want to add other insights.

I am going to demonstrate that the problem in our installation starts when updating to Elementor v.1.6.0, when was introduced “Tweak! - The editor loading now in the wp-admin area instead of the front-end” 

I compared the working site installed version plugins with the actual with problems.
Working version had Elementor v.1.5.5 and Elementor Pro v1.5.5.
The staging site with problems has Elementor v.1.6.4 and Elementor Pro v.1.5.8.

I rolled back the two plugins, regenerated the files (Elementor -> Tools -> General), cleared all the cache.
Elementor editor is working, it has lost the icons when hovering on the various sections/widgets, clearing browser cache did not helped, but tooltips on hover are working, and the Page Settings are working fine, as expected.
Then I proceeded updating the plugins respecting their release date order, each time regenerating the files and clearing the cache, with this results:

2017-07-12 v1.5.6
no change
2017-07-24 v1.5.7
no change
2017-07-25 v1.5.8
no change
1.6.0 - 02.08.2017
obviously the Elementor editor opens now in the wp-admin.
The problem appears here.
Since the "Buggy" title is related to a product page I temporary disabled WooCommerce but it still get stuck on that page, although it returns 404 Page not found. It is clear that the product page still exists in the database and it is not my intention to delete it, it is part of the live site, and it must continue to exist.
In another attempt made before writing this second report, I also tried changing product permalinks, which is a custom one, to the default one, but with no change, it simply opened the product page with the actual "/product/" structure as per default permalinks settings.


I skipped the other updates because the problem starts from v.1.6.0 and it is still present on latest v.1.6.4
1.6.1 - 02.08.2017

1.6.2 - 03.08.2017

1.6.3 - 09.08.2017

1.6.4 - 06.08.2017	(the mistake on the date is on the GitHub page https://github.com/pojome/elementor not mine)

Then I made another test building a brand new site with similar characteristics and found no problem.
So it is possible to consider this as related to the complex installation and not a bug.

I leave this as a trace in case other encounter similar problems.
I will continue working on its solution after the next week, and update this issue when fixed.
